boms/src/main/resources/m2repo/README.md

```diff
@@ -0,0 +1,5 @@
+WildFly Swarm Maven Repository
+==============================
+
+This archive contains Maven repository artifacts for WildFly Swarm ${project.version}
+using the ${project.artifactId} BOM.
```

Comment:
Maybe also add the groupId, like `${project.groupId}:${project.artifactId}`? Otherwise the end result might be e.g. "using the bom BOM", which looks weird.

Reply:
Good point, I'll follow on since this is merged.